src/components/Menu/MenuItem/menuItem.module.scss

```diff
@@ -12,12 +12,6 @@
   margin: $space-xxs 0;
   white-space: nowrap;
 
-  &:focus-within {
```

Comment:
Was this causing any visual issue when keyboard is navigated to it?

Reply:
Yes. If you see and the end of the file, we have global focus visible = which is contextprovider driven